backend/src/routes/admin/flag_captures/list.rs

```diff
@@ -0,0 +1,65 @@
+use crate::entities::flag_capture;
+use crate::utils::app_state::AppState;
+use crate::utils::error::Error;
+use actix_web::web::{Data, Path};
+use actix_web::{HttpResponse, get};
+use sea_orm::ColumnTrait;
+use sea_orm::EntityTrait;
+use sea_orm::QueryFilter;
+use uuid::Uuid;
+
+#[utoipa::path(
+    responses(
+        (status = 200, description = "Flag captures successfully fetched.", body = Vec<flag_capture::Model>),
+        (status = 500, description = "Internal server error.")
+    ),
+    tag = "admin/flag_captures"
+)]
+#[get("/list")]
+pub async fn list(app_state: Data<AppState>) -> Result<HttpResponse, Error> {
+    let flag_captures = flag_capture::Entity::find()
+        .all(&app_state.database)
+        .await?;
+
+    Ok(HttpResponse::Ok().json(flag_captures))
+}
+
+#[utoipa::path(
+    responses(
```

Comment:
In future I'll create a new system for writing admin endpoints but you could try playing with
```rs
#[derive(Deserialize)]
pub struct ListQuery {
    team_id: Option<Uuid>,
    task_id: Option<String>,
}
```

And building queries based on the params...